src/interaction.js

```diff
@@ -178,7 +178,7 @@ export const executeScript = async (...props) => {
     try {
       const extractor = extractParameters("script")
       const {code, args, limit} = await extractor(props)
-
+      console.log(`About to run extractor with props: ${JSON.stringify(props)}`)
```

Comment:
Adding this log statement here fixes the issue. Not sure why this is, possibly something to with bundling/minified version??